package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "desafio-taqtile",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
```

Geralmente aqui na instaq, deixamos as dependências com versoes fixas, assim vc evita sofrer com breaking changes. Altere nas outras também.
```suggestion
    "react": "18.3.1",
```